### PR TITLE
Add option timestamp first packet

### PR DESCRIPTION
**Pull request by wep21**
ros2 porting for #283
a part of https://github.com/ros-drivers/velodyne/pull/434

### CODE REVIEW

**Comment by wep21** [comment]
cc @clalancette @JWhitleyWork

File: velodyne_driver/include/velodyne_driver/driver.hpp [hunk 1]
```diff
@@ -71,6 +71,7 @@ class VelodyneDriver final : public rclcpp::Node
     int cut_angle;                   // cutting angle in 1/100°
     double time_offset;              // time in seconds added to each velodyne time stamp
     bool enabled;                    // polling is enabled
+    bool timestamp_first_packet;
```
Comment (clalancette) [inline velodyne_driver/include/velodyne_driver/driver.hpp:74]: Nit: can you please add a comment to this line, to be consistent with the rest of the structure?

**Review by clalancette — CHANGES_REQUESTED** [review]
Generally looks good to me, I'm happy to approve and merge with the one nit fixed.

File: velodyne_driver/include/velodyne_driver/driver.hpp (hunk 1 above)
Comment (wep21) [reply velodyne_driver/include/velodyne_driver/driver.hpp]: @clalancette add comment at https://github.com/ros-drivers/velodyne/pull/436/commits/c78129f23ed448b8a0a2fa6dc1514b577aeed331

**Review by clalancette — APPROVED** [review]

**Pull request merged**